reframe/frontend/cli.py

```diff
@@ -178,11 +179,16 @@ def main():
     misc_options.add_argument(
         '--system', action='store',
         help='Load SYSTEM configuration explicitly')
+
+    filepath = pathlib.Path(__file__)
```

Comment:
I know you prefer the pathlib, but for consistency inside the framework, I suggest using `os.path`. I suggest writing it also in a single-line. You could perhaps define the default settings file as a global variable `DEFAULT_SETTINGS_FILE = os.path.abspath('../../reframe/settings.py')` inside the `cli.py`.